soil-service/spec/services/fields/humus_balance_service_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Fields::HumusBalanceService do
+  describe '#call' do
+    subject { described_class.new(crops: crops).call }
+
+    let(:crops) { [1, 8, 3] }
+
+    it 'calculate humus balance' do
+      expect(subject).to eq(19.11)
```

Comment:
Nice that you have tests!

But there should be more of them to cover more cases.